internal/website/content/howto/secrets/open-keeper.md

```diff
@@ -73,6 +73,30 @@ obtain [GCP credentials][] and then create a gRPC connection to GCP KMS.
 [GCP credentials]: https://cloud.google.com/docs/authentication/production
 [`gcpkms.OpenKeeper`]: https://godoc.org/gocloud.dev/secrets/gcpkms#OpenKeeper
 
+## Azure KeyVault {#azure}
+
+The Go CDK can use keys from [Azure KeyVault][] to keep information secret.
+`secrets.OpenKeeper` will use [default credentials from the environment][Azure
+Environment Auth], unless you set the environment variable
+`AZURE_KEYVAULT_AUTH_VIA_CLI` to `true`, in which case it will use
+credentials from the `az` command line.
```

Comment:
... from the `az` CLI configuration? Or ... from `az configure`?

Reply:
It's from whatever `az` stores when you do `az login`.